src/migrations/1752100192730-CleanupUserPreferencesConstraints.ts

```
@@ -0,0 +1,53 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
```

ESLint

[error] Replace `"typeorm"` with `'typeorm'`

(prettier/prettier)